test/recipes/90-test_store.t

```diff
@@ -89,43 +101,71 @@ indir "store_$$" => sub {
 
         foreach (@noexist_files) {
             my $file = srctop_file($_);
-            ok(!run(app(["openssl", "storeutl", $file])));
-            ok(!run(app(["openssl", "storeutl", to_abs_file($file)])));
-            ok(!run(app(["openssl", "storeutl", to_abs_file_uri($file)])));
+        SKIP: {
+                ok(!run(app(["openssl", "storeutl", $file])));
+                ok(!run(app(["openssl", "storeutl", to_abs_file($file)])));
```

Comment:
Any particular reason why these (and other similar instances below) are inside the SKIP block?

Reply:
Estetics.  I found it nicer looking of all ok lines were inside SKIP blocks